package.json

```diff
@@ -90,8 +90,7 @@
   "lint-staged": {
     "*.{md,yml,json}": "prettier --write",
     "*.{js,ts}": [
-      "eslint --fix",
-      "prettier --write"
```

Comment:
Not needed, as this is handled through eslint. We can now disable prettier as needed.